src/InstallWizard.js

```diff
@@ -20,40 +20,19 @@ import WizardProgress from './components/WizardProgress';
 import { LoadingMask } from './components/LoadingMask.js';
 import { Map, fromJS } from 'immutable';
 import { fetchJson, postJson, deleteJson } from './utils/RestUtils.js';
-
-// global states vars used in installation and update
-const COMMON_GLOBAL_STATES_VARS = [
-  'playbookStatus', 'model', 'connectionInfo', 'deployConfig', 'isUpdateMode'
-];
-
-// global states vars used in update
-const UPDATE_GLOBAL_STATES_VARS = [
-  'currentMenuName', 'processMenuName', 'processOperation', 'steps', 'pages', 'operationProps',
-  'wizardLoadingErrors', 'wizardLoading'
-];
-
-// global persisted states vars used in installation and update
-const COMMON_PERSISTED_STATES_VARS = [
-  'currentStep', 'steps', 'playbookStatus', 'connectionInfo', 'deployConfig'
-];
-
-// global persisted states vars used in update
-const UPDATE_PERSISTED_STATES_VARS = [
-  'currentMenuName', 'processMenuName', 'processOperation', 'operationProps'
-];
+import {
```

Comment:
I would prefer that these arrays not be put into `constants.js`, since they are very specific to the internal workings of this class and should not be exposed elsewhere. Furthermore, I don't even think they need to be constants since  they are each used only one line in the class (in the constructor).

Reply:
k, will look into it.